src/utils/iqeEnablement.ts

```diff
@@ -19,6 +19,8 @@ const AUTH_ALLOWED_ORIGINS = [
   /https:\/\/api(?:\.[a-z]+)?\.openshift(?:[a-z]+)?\.com/,
   /https:\/\/api?\.demo-experience(?:\.[a-z]+)?\.demo?\.redhat?\.com/,
   /https:\/\/api?\.aws?\.ap-southeast-1(?:\.[a-z]+)?\.openshift?\.com/,
+  /https:\/\/console-service?\.[a-z]*-[a-z]*\.aws?\.ansiblecloud?\.com\/api/,
+  /https:\/\/console-service?\.[a-z]*-[a-z]*\.aws?\.ansiblecloud?\.redhat?\.com\/api/,
```

Comment:
@acosferreira a few questions
1. Do we have any documentation surrounding these domains? Such as who owns them, what APIs are surfaced, etc? I think this might help clear up some of my additional questions below and help me understand any security implications with sending auth tokens to these endpoints.
2. If these API's are accessed through AWS (guessing by the URL) - are we sure `[a-z]*-[a-z]*` is sufficient for the region? I believe regions may also contain numerics
3. Is the `.redhat` portion of the URL truly optional? If so - do we need 2 regex's or can we simplify to just one where `.redhat` is an optional non-capturing group in the regex (which should catch both cases) IE: 
```suggestion
  /https:\/\/console-service?\.[a-z]*-[a-z]*\.aws?\.ansiblecloud?(?:\.redhat)?\.com\/api/,
```

FWIW this is just a global helper to append the token to HTTP Fetch calls made to known-safe URLs. Nothing is stopping you from sending the token in your application without these changes.

Reply:
Thanks for the reply @florkbr.

Those aws are part of ansible-as-service project and  `[a-z]*-[a-z]*`  represents our mgt-env (dev, int ,preprod, prod), not the regions.
For our dev and int environments we are not using .redhat.com, only for preprod and prod, Im also adding a validation to our int env because is ` https://console-service.mgt-int.int.aws.ansiblecloud.com/api/v1/subscriptions/ `